set-quotas.py

```diff
@@ -100,6 +100,11 @@ def parse_rows(rows, select_project=None):
             for quota_name, value in quotas.iteritems():
                 quotas[quota_name] = int(value)
 
+            # Initial port padding based off of instance and network amount.
+            if 'instances' and 'port' and 'networks' in quotas:
```

Comment:
nope, sorry for not catching it earlier.
all(k in quotas for k in ['instances', 'port', 'networks'])

Reply:
All set now.